test/testConfig.test.js

```diff
@@ -90,6 +92,19 @@ describe('user configuration', function () {
     expect(config.getTempPasswordConfig()).to.be.eql(defaultSettings.tempPassword);
   });
 
+  it('should override default settings for SSL certificate', function () {
+    const user = {
+      sslKeyPemPath: "my-key.pem",
+      sslCertPemPath: "my-cert.pem"
+    };
+    fs.writeFileSync(tempUserFile, JSON.stringify(user));
+
+    const config = require('../src/config');
+
+    expect(config.getSSLKeyPath()).to.be.eql(user.sslKeyPemPath);
+    expect(config.getSSLCertPath()).to.be.eql(user.sslCertPemPath);
+  });
+
```

Comment:
Can you confirm? This confirms that some defined key and certificate paths are loaded correctly into config?